frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx

```diff
@@ -92,6 +99,21 @@ export default function Values({
     });
   });
 
+  function fixSmallBarChartValues(
+    singleSeriesValues: Value[],
+    numberOfBarSeries: number,
+  ) {
+    const barWidth = innerBarScale?.bandwidth() ?? Infinity;
+    const isBarTooSmall = barWidth < 20;
+    // Use the same logic as in https://github.com/metabase/metabase/blob/cb51e574de31c7d4485a9dfbef3261f67c0b7495/frontend/src/metabase/visualizations/lib/chart_values.js#L138
+    if (numberOfBarSeries > 1 && isBarTooSmall) {
```

Comment:
Basically the same logic as
https://github.com/metabase/metabase/blob/cb51e574de31c7d4485a9dfbef3261f67c0b7495/frontend/src/metabase/visualizations/lib/chart_values.js#L138

but I don't think we need to rotate the values 90 degrees like in the viz app yet so we can keep it simple. We could handle rotating the values if there's a use case for that as we would only show the rotated values when **Values to show** is set to `all` and we don't even pass that option to static chart.
e.g. 
![image](https://user-images.githubusercontent.com/1937582/189692696-5e3a68a7-92a4-431b-a44a-a68113ba2605.png)